journal/202307.md

```diff
@@ -0,0 +1,596 @@
+- Explicitly configured RPC clients connecting to dcrd via HTTP to [use TLS v1.2 or higher](https://github.com/decred/dcrd/pull/3169). This was already the implicit minimum, but now it is more clear.
+
+Developer and internal changes merged in `master`:
+
+- Updated the comments detailing the [ASERT calculation coefficients](https://github.com/decred/dcrd/pull/3156) in the mining code so that they match the consensus code comments.
+- Added a [new linter](https://github.com/decred/dcrd/pull/3157), addressed some [linter complaints](https://github.com/decred/dcrd/pull/3155), and moved [linting logic](https://github.com/decred/dcrd/pull/3161) to its own script outside of the testing script.
+- Updated [Github Actions](https://github.com/decred/dcrd/pull/3159), and ensured that the continuous integration actions happen in the [correct order](https://github.com/decred/dcrd/pull/3166).
+- Updated [golangci linter](https://github.com/decred/dcrd/pull/3163), and changed its [method of installing](https://github.com/decred/dcrd/pull/3162) from `curl` to `go install`, which has the benefit of reducing potential security risks from supply-chain attacks. Also cached the binary for faster future runs.
+- Current versions of Go have a confusing way of handling loops. Both "for loops" and "range loops" treat internal loop variables ambiguously, which has led to code which is difficult to read and tricky to debug. Devs have had to treat loops with an abundance of caution, yet still potentially deal with annoying errors. A full explanation of the problem can be [read here](https://go.googlesource.com/proposal/+/master/design/60078-loopvar.md), especially the section which talks about [the rationale](https://go.googlesource.com/proposal/+/master/design/60078-loopvar.md#rationale-and-compatibility). Future versions of Go (likely 1.22) will introduce a breaking change to how loops are handled. While the new loop semantics will be less confusing and error-prone, projects will have to set a very high minimum supported Go version to avoid bugs when compiled with older Go. To ensure the dcrd code will work correctly with Go 1.22, as well as remain backwards-compatible and compile correctly with older versions of Go, [all necessary loops were updated](https://github.com/decred/dcrd/pull/3165). This will avoid any potential issues before they even have a chance to arise.
+
+
+### dcrwallet
+
+_[dcrwallet](https://github.com/decred/dcrwallet) is a wallet server used by command-line and graphical wallet apps._
+
+The following work has been merged in `master` towards future releases.
+
+The primary improvements users may notice are related to purchasing tickets through a VSP:
```

Comment:
Thanks for spotting this!

Since it was merged in June it should have been in the [June issue](https://xaur.github.io/decred-news/journal/202306) but we missed it. Actually we added it for consideration in 438af24 but I could not figure out the scope of the the bug and the fix, assumed it is one of those smaller context fixes/refactors, and removed it in 38d1d2d.

Added a massaged version of your comment as a first bullet. Thanks for explaining how it works.

Reply:
Thanks for checking. Updated.